tools/pkg-config/stumpless.pc.in

```diff
@@ -0,0 +1,10 @@
+prefix=@CMAKE_INSTALL_PREFIX@
+exec_prefix=${prefix}
+libdir=${exec_prefix}/lib
+includedir=${prefix}/include
+
+Name: @CMAKE_PROJECT_NAME@
+Description: C logging library built for high performance and a rich feature set
+Version: @CMAKE_VERSION@
```

Comment:
I believe this should be `@CMAKE_PROJECT_VERSION@`? As is the version of the package is set to the version of CMake used in the configure step.

Reply:
.. yep! Good catch! Copy and pasted from Collections-C, applies there as well. Will contribute there too. Should've been [CMAKE_PROJECT_VERSION](https://cmake.org/cmake/help/latest/variable/CMAKE_PROJECT_VERSION.html).

Reply:
Yeah no, could up the requirement to 3.12 though? Which was released back in '18, so any distro/package manager should be shipping that by default now.

Reply:
Updated the minimum requirement.

Reply:
I'd like to avoid updating the minimum requirement until the next major version release, as this could break existing builds. For example, my WSL install only has 3.10.2 in it's package manager.

This does need to be updated though, and I think this is a good indicator that needs to happen soon. I've opened #341 to track this, since there are other improvements I'd like to make in the commit to increase the requirement.

For this particular pull request, please switch to [PROJECT_VERSION](https://cmake.org/cmake/help/v3.2/variable/PROJECT_VERSION.html), and I'll update it in the resolution to #341.